pkg/client/dbwrapper.go

```diff
@@ -165,3 +165,15 @@ func (db DbWrapper) BatchDelete(ctx context.Context, table string, keys []string
 	}
 	return nil
 }
+
+func (db DbWrapper) Analyze(ctx context.Context, table string) (err error) {
+	analyzeDB, ok := db.DB.(ycsb.AnalyzeDB)
+	if ok {
+		start := time.Now()
```

Comment:
I don't think we need to measure this operation. If we want ,we can use ANALYZE directly, not `ANALYZE TABLE`

Reply:
Okay